node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Deze hele zou dir zou in de .gitignore moeten zitten, en deze file zou dus niet gecommit moeten zijn.

Reply:
Yep, add to .gitigore